base.py

```diff
@@ -367,6 +368,8 @@ def create_connect_args(self, url):
             opts['port'] = '443'
         opts['autocommit'] = False  # autocommit is disabled by default
         opts.update(url.query)
+        self._cache_column_metadata = opts.get('cache_column_metadata',
```

Comment:
nit: If `cache_column_metadata` is not a standard sqlalchemy option, it may be helpful to others if the option was documented somewhere.

Reply:
Yes, if the cache works well, I'll update the doc too.